library/tasks/FastqToUBam.wdl

```diff
@@ -11,7 +11,14 @@ task FastqToUBam {
   Int estimated_disk_required = ceil(size(fastq_file, "G") * 2.2)
 
   command {
-    mv "${fastq_file}" "${fastq_file}""${fastq_suffix}"  # add suffix; does nothing if not provided
+    set -e
+
+    # Adds fastq_suffix if it is passed
+    if [ ! -z "${fastq_suffix}" ];
```

Comment:
instead of this bash you could do
`String? fastq_suffix` in your inputs and
`{"mv " + fastq_file + " " + fastq_file + fastq_suffix}`

not even sure if this is better but just a different option

Reply:
If you move a file to the same place it emits a warning to stderr, which is why I added the bash.

Reply:
ah but this will only actually write out the `mv` command to the script if `fastq_suffix` is defined

Reply:
That's true, but we define a default parameter of "", so it is always defined, but the empty string fails the `-z` command.